oracle/tests/test_custom_queries.py

```diff
@@ -0,0 +1,126 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import mock
+import pytest
+
+from datadog_checks.base.utils.db import QueryManager
+from datadog_checks.oracle import Oracle
+
+from .common import CHECK_NAME
+
+pytestmark = pytest.mark.unit
+
+
+def test__check_only_custom_queries(instance):
+    """
+    Test the default metrics are not called when only_custom queries set to true
+    """
+    instance['only_custom_queries'] = True
+
+    check = Oracle(CHECK_NAME, {}, [instance])
+
+    assert check._query_manager.queries == []
+
+
+def test__check_only_custom_queries_not_set(instance):
+    """
+    Test the default metrics are called when only_custom queries is not set
+    """
+    instance['only_custom_queries'] = False
+
+    check = Oracle(CHECK_NAME, {}, [instance])
+
+    assert check._query_manager.queries != []
```

Comment:
Nit- can we make this one test and parameterize it?

Reply:
Following up in a separate PR https://github.com/DataDog/integrations-core/pull/11902